kserve/runtimes/24.05-llama3-70b-instruct.yaml

```diff
@@ -27,13 +27,6 @@ spec:
     ports:
     - containerPort: 8000
       protocol: TCP
-    resources:
-      limits:
```

Comment:
Why are we removing this without adding new sane defaults?

Reply:
@supertetelman the limits you defined is not necessary but doesn't calculate how much it needs and I have seen the problem that if the system doesn't have enough resource what we defined it will be pending state

Reply:
A pending state is preferable to a NIM starting up without enough performance. The better thing to do would be to set these defaults to the minimum recommended for each NIM;  I am working with another team to get those values published.